vector/src/main/res/values/styles.xml

```diff
@@ -39,6 +39,11 @@
         <item name="android:textSize">16sp</item>
         <item name="android:dropDownHorizontalOffset">0dp</item>
         <item name="android:dropDownVerticalOffset">0dp</item>
+        <!--
+            Before Kitkat the popup background is white on dark theme, so force color here.
```

Comment:
You should create a Vector.PopupMenuBase to avoid duplicate attribute between this and the v21 version. We can discuss this IRL